tests/pytest/test_dft_benchmarks.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+
+import pytest
+from utils import *
+from addons import *
+
+import psi4
+
+
+@pytest.fixture
+def dft_bench_systems():
+    ang = np.array([
+      [ -1.551007,  -0.114520,   0.000000],
+      [ -1.934259,   0.762503,   0.000000],
+      [ -0.599677,   0.040712,   0.000000]])
+    oldang = ang * 0.52917721067 / 0.52917720859
+    oldmass = [15.99491461956, 1.00782503207, 1.00782503207]
+
+    h2o = psi4.core.Molecule.from_arrays(geom=oldang, elez=[8, 1, 1], units='Angstrom', mass=oldmass)
+    h2o_plus = psi4.core.Molecule.from_arrays(geom=oldang, elez=[8, 1, 1], units='Angstrom', mass=oldmass,
+                                              molecular_charge=1, molecular_multiplicity=2)
+
+    h2o_dimer = psi4.geometry("""
+        0 1
+        O  -1.551007  -0.114520   0.000000
+        H  -1.934259   0.762503   0.000000
+        H  -0.599677   0.040712   0.000000
+        --
+        0 1
+        O   1.350625   0.111469   0.000000
+        H   1.680398  -0.373741  -0.758561
+        H   1.680398  -0.373741   0.758561
+        no_reorient
+        """)
+
+    psi4.set_options({
+       'dft_radial_points': 200,
+       'dft_spherical_points': 590,
+       'guess': 'sad',
+       'e_convergence': 9,
+       'd_convergence': 9,
+    })
+
+    return {'h2o': h2o, 'h2o_plus': h2o_plus, 'h2o_dimer': h2o_dimer}
```

Comment:
Maybe `yield` these then set the above `set_options` as an opt stash to be cleared after the yield returns?

Reply:
Not quite following. All options get cleared btwn tests via conftest and a fixture gets rerun for every parametrize, so what aspect are you seeking to change?

Reply:
Oh, if options are always nuked between tests then no comment from me. I was thinking about manually handling that via fixture cleanups.

Reply:
yup, https://github.com/psi4/psi4/blob/master/tests/pytest/conftest.py#L9-L14. iirc, `clean_options` was invented for pytest, as we were leaking options madly.